### PR TITLE
library added to about me java file

### PR DESCRIPTION
**Pull request by criticalAY**
have updated the library in the about me activity
issue #1002

### CODE REVIEW

File: app/src/main/java/protect/card_locker/AboutActivity.java [hunk 1]
```diff
@@ -62,6 +62,7 @@ protected void onCreate(Bundle savedInstanceState) {
         }
 
         final List<ThirdPartyInfo> USED_LIBRARIES = new ArrayList<>();
+        USED_LIBRARIES.add(new ThirdPartyInfo("Yalantis(Image Cropping)","com.github.yalantis:ucrop","Apache 2.0"));
```
Comment (TheLastProject) [inline app/src/main/java/protect/card_locker/AboutActivity.java:65]: Yalantis is the developer, not the library name. It's called uCrop

com.github.yalantis:ucrop is not a valid URL, the library is at https://github.com/Yalantis/uCrop.

**Review by TheLastProject — CHANGES_REQUESTED** [review]

File: app/src/main/java/protect/card_locker/AboutActivity.java (hunk 1 above)
Comment (criticalAY) [reply app/src/main/java/protect/card_locker/AboutActivity.java]: file updated

**Comment by criticalAY** [comment]
by mistake, one other file was also committed which was for language config. Should I delete that file in here?

**Comment by Altonss** [comment]
> by mistake, one other file was also committed which was for language config. Should I delete that file in here?

Yes it's probably better, you could open another PR for better visibility :)

**Comment by criticalAY** [comment]
> > by mistake, one other file was also committed which was for language config. Should I delete that file in here?
> 
> Yes it's probably better, you could open a separate PR for the language config for better visibility :)

ok cool

File: app/src/main/java/protect/card_locker/AboutActivity.java [hunk 2]
```diff
@@ -62,6 +62,7 @@ protected void onCreate(Bundle savedInstanceState) {
         }
 
         final List<ThirdPartyInfo> USED_LIBRARIES = new ArrayList<>();
+        USED_LIBRARIES.add(new ThirdPartyInfo("uCrop","https://github.com/Yalantis/uCrop.","Apache 2.0"));
```
Comment (TheLastProject) [inline app/src/main/java/protect/card_locker/AboutActivity.java:65]: There's an extra . at the end of the URL that shouldn't be there

**Review by TheLastProject — CHANGES_REQUESTED** [review]

File: app/src/main/java/protect/card_locker/AboutActivity.java (hunk 2 above)
Comment (criticalAY) [reply app/src/main/java/protect/card_locker/AboutActivity.java]: removed :)

**Comment by criticalAY** [comment]
removed the dot

**Comment by criticalAY** [comment]
@TheLastProject i have done the changes can PR be approved now?

**Pull request merged**